localstack-core/localstack/utils/container_utils/docker_cmd_client.py

```diff
@@ -909,12 +909,15 @@ def _check_and_raise_no_such_container_error(
         if any(msg.lower() in process_stdout_lower for msg in error_messages):
             raise NoSuchContainer(container_name_or_id, stdout=error.stdout, stderr=error.stderr)
 
-    def _transform_container_labels(self, labels: str) -> Dict[str, str]:
+    def _transform_container_labels(self, labels: str | Dict[str, str]) -> Dict[str, str]:
```

Comment:
Could you please change this to a Union[]? This class is used by the CLI, where we have a lower python version bound of 3.9 currently, which does not support the pipe operator as type union yet.

Reply:
Should be addressed now 👍🏻